src/analysis/normed_space/star/spectrum.lean

```diff
@@ -0,0 +1,32 @@
+/-
+Copyright (c) 2022 Jireh Loreaux. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Jireh Loreaux
+-/
+import analysis.normed_space.star.basic
+import analysis.normed_space.spectrum
+
+/-!
+In this file, we establish various propreties related to the spectrum of elements in a
+C⋆-algebra over ℂ.
+-/
+
+variables {A : Type*}
+[normed_ring A] [normed_algebra ℂ A] [star_ring A] [cstar_ring A] [complete_space A]
+[measurable_space A] [borel_space A] [topological_space.second_countable_topology A]
+
+open_locale topological_space ennreal
+open filter ennreal
+
+lemma spectral_radius_eq_nnnorm_of_self_adjoint {a : A} (ha : a ∈ self_adjoint A) :
+  spectral_radius ℂ a = ∥a∥₊ :=
```

Comment:
Same thing here: a version for `x : self_adjoint A` would be nice to have as well, maybe called `self_adjoint.coe_spectral_radius_eq_nnnorm` or something like that.

Reply:
done